Library/Notifier.swift

```diff
@@ -20,19 +20,29 @@ struct Notifier {
         let notificationID = dateFormatter.string(from: date)
         let sunriseDate = location.sunriseForDate(date)
 
+        let formattedMessage = self.formattedMessage(location: location, date: date)
+
+        print(formattedMessage)
+
+        // TODO: Fix deprecated UILocalNotification
+        UILocalNotification.schedule(notificationID, at: sunriseDate, message: formattedMessage)
+    }
+
+    static func formattedMessage(location: Location, date: Date) -> String {
         let interval = location.dayLengthDifferenceOnDate(date)
 
         let messageGenerator = MessageGenerator()
 
-        let minutesRounded = abs(Int(Darwin.round(interval / 60.0)))
-        let generatedMessage = messageGenerator.messageForNotification(forDate: Date(), yesterdayDaylightLength: location.yesterdayDaylightLength, todayDaylightLength: location.todayDaylightLength, tomorrowDaylightLength: location.tomorrowDaylightLength)
+        // never say 0 minute…
```

Comment:
Please expand on this comment 🙏  

Probably should say something like "We don't wanta minute value of 0 in our message, so we use 1 as the minimum value" (or something)